src/mpid/ch4/include/mpidpre.h

```diff
@@ -25,6 +25,8 @@
 #include "mpl_uthash.h"
 #include "ch4_coll_params.h"
 
+extern MPIDI_CH4_coll_algo_container_t mpir_fallback_container;
```

Comment:
@AlexeyMalkhanov didn't catch this in the last round, but can you rename to `MPIDI_coll_algo_container_t`? Thanks.